src/amo/components/StaticPages/About/index.js

```diff
@@ -39,8 +39,8 @@ export class AboutBase extends React.Component {
               // eslint-disable-next-line react/no-danger
               dangerouslySetInnerHTML={sanitizeHTML(
                 i18n.sprintf(
-                  i18n.gettext(`You can also use AMO to find add-ons for Mozilla
-                      %(startTBLink)sThunderbird%(endTBLink)s and
+                  i18n.gettext(`If you are looking for add-ons for Thunderbird or SeaMonkey, please visit
+                      %(startTBLink)sThunderbird%(endTBLink)s or
                       %(startSMLink)sSeaMonkey%(endSMLink)s.`),
```

Comment:
And here please change `SeaMonkey` to `addons.thunderbird.net/seamonkey` As @caitmuenster suggested [here](https://github.com/mozilla/addons-frontend/issues/5859#issuecomment-411090639)